media_player/alexa.py

```diff
@@ -979,6 +998,32 @@ def _get_request(self, uri, data=None):
                 message))
             return None
 
+    def get_last_device_serial(self):
+        """Identify the last device's serial number."""
+        try:
+            response = self._get_request('/api/activities?startTime=&size=1&offset=1')
+        except Exception as ex:
+            template = ("An exception of type {0} occurred."
+                        " Arguments:\n{1!r}")
+            message = template.format(type(ex).__name__, ex.args)
+            _LOGGER.error("An error occured accessing the API: {}".format(
+                message))
+            return None
+        try:
+            response.json()
```

Comment:
Within the try clause, let's save the json portion we need to a variable and use the variable later in lines 1022, 1023. This will help readability and also should catch a potential exception if the json array is empty.  For example something like:
```suggestion
            last_activity = response.json()['activities'][0]
```
Please review the suggested variable name; I'm not certain what's the best way to describe it.

Reply:
Good idea. It should make it a bit easier to follow the json tree as well. Thank you.

Reply:
I mean let's move it into the first try block. No need to duplicate the exception catching code.